src/app/dataset-block/metadata-block/metadata-block.component.ts

```diff
@@ -58,9 +69,30 @@ export class MetadataBlockComponent
         );
     }
 
+    public onPageChange(params: {
+        currentPage: number;
+        isClick: boolean;
+    }): void {
+        this.loadHistory(params.currentPage - 1);
+    }
+
     private loadMetadataBlock(): Subscription {
         return this.blockService
             .requestMetadataBlock(this.datasetInfo, this.blockHash)
             .subscribe();
     }
+
+    private loadHistory(page = 0) {
```

Comment:
No data type for `page` argument and return type

Reply:
Eslint doesn't let me specify type manually for method argument. Fixed for return type